### PR TITLE
Added Lidarr support to Upgradinatorr

### PR DESCRIPTION
**Pull request by PearsonFlyer**
Upgradinatorr modified to add Lidarr support, docs and config template also modified.

### CODE REVIEW

File: Upgradinatorr/upgradinatorr.ps1 [hunk 1]
```diff
@@ -40,6 +41,9 @@ function Add-Tag {
         $series,
         [Parameter()]
         [int]
+        $artist,
```
Comment (RobinDadswell) [inline Upgradinatorr/upgradinatorr.ps1:44]: Make artist an array like movies and series not a an int :)

Comment (PearsonFlyer) [reply Upgradinatorr/upgradinatorr.ps1]: I have done this, and it's gotten rid of the warning on tagging. Thanks!

Comment (RobinDadswell) [reply Upgradinatorr/upgradinatorr.ps1]: No problem at all :)

**Comment by PearsonFlyer** [comment]
Sorry, my newest one fixes the artist name in the output. Have confirmed everything works as expected - tagging, searching, and output.

**Comment by angrycuban13** [comment]
@PearsonFlyer it is all working correctly then?

**Comment by PearsonFlyer** [comment]
Yep, ready to go.

**Comment by angrycuban13** [comment]
We forgot to update the README.md lol

**Comment by angrycuban13** [comment]
@PearsonFlyer I'm going to push an update shortly, can you check the parameters under `Lidarr Configuration Attributes` in README.md and update them as needed?

**Comment by PearsonFlyer** [comment]
I will take care of those as well. I noticed it earlier this afternoon.

**Comment by angrycuban13** [comment]
Go wild @PearsonFlyer

**Pull request merged**